build/run-code-lint.sh

```diff
@@ -22,7 +22,7 @@ gem install mdl
 gem install awesome_bot
 
 # Install golangci-lint
-curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)"/bin v1.46.2
```

Comment:
It may be safer to remain go linter until there is any breaking change. The newer version of the go linter always reports some new un-expected errors :-)

Reply:
The upgrade was actually because of a breaking change with the current release
Error:
```
panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.22()
        github.com/go-critic/go-critic@v0.6.3/checkers/embedded_rules.go:47 +0x4b4```
Upgrading to the latest release and disabling a few new linters, solves the issue.

Reply:
got it, thanks!